_extensions/debruine/glossary/glossary.yml

```
@@ -0,0 +1,22 @@
+data: |
+  Data can mean any piece of information that some person or people want to use
+  to answer questions. What is considered data (and metadata) is highly dependent
+  on people and what they intend to do with information that is collected.
+  In the context of Seedcase Sprout, data is any information collected for the
+  purposes of doing analyses on them to answer questions. An example might be
+  data collected from people participating in a study on health and disease.
+data package: |
+  A "container" for data that describes a coherent collection of data. It consists
+  of two parts: data resources and properties.
+data resource: |
+  A single piece of data and metadata that is part of a data package. It contains
+  the actual data that is collected saved and documented following the Data Package
```

```suggestion
  the actual data that is collected and saved, documented following the Data Package
```
Maybe? To avoid making it sound like it's collected and saved following the Data Package standard

---

Or is it the case that a comma is missing? 'collected, saved, and documented' Otherwise I'd put a 'then' into the correction, like this: 'is collected and saved, then documented following the Data Package standard.'

---

Sure, those sound fine to me too! Maybe the missing comma version makes the most sense to me.

---

I removed "collected and saved" to keep it simpler. What do you think? :)

---

I like that solution.